test/remote/gateways/remote_blue_snap_test.rb

```diff
@@ -139,6 +141,13 @@ def test_failed_purchase
     assert_equal '14002', response.error_code
   end
 
+  def test_failed_purchase_with_invalid_cabal_card
```

Comment:
Can we test the success case for the cabal card type? I saw we added `@cabal_credit_card` but didn't find it used anywhere in our remote test.

Reply:
Nevermind on that. Re-read your PR description and see you did try to do that. 🤦‍♂